<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskPanel</class>
 <widget class="QWidget" name="TaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add object</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="props_group">
     <property name="toolTip">
      <string>The tomography properties. The larger the number of projections, the larger the computation time. Along the same line, the lower the error the longer computations.</string>
     </property>
     <property name="title">
      <string>Tomography parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="angle_group">
        <property name="toolTip">
         <string>The number of angluar projections. More projections result in better quality images. However, each angular projection requires a full radiography, increasing the required time to compute the full tomography</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="dens_group_kk">
         <item>
          <widget class="QLabel" name="angles_label">
           <property name="text">
            <string>Projection angle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="angles">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>720</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>180</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="halt_group">
        <property name="toolTip">
         <string>The maximum admisible error. The smaller this value the longer will it take to finish</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="max_error_label">
           <property name="text">
            <string>Maximum error</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="max_error">
           <property name="text">
            <string notr="true">0.2</string>
           </property>
           <property name="unit" stdset="0">
            <string notr="true"/>
           </property>
           <property name="quantity" stdset="0">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="results_group">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QProgressBar" name="pbar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="image_group">
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="4" column="1">
          <widget class="QComboBox" name="cmap">
           <item>
            <property name="text">
             <string>Radiography</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Radiography negative</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Airport</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="image">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="image_label">
           <property name="text">
            <string>Show Image</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="cmap_label">
           <property name="text">
            <string>Color map</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="crange_label">
           <property name="text">
            <string>Color range</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="slice_label">
           <property name="text">
            <string>Slice</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="slice">
           <property name="pageStep">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="run">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="use_gpu">
        <property name="toolTip">
         <string>Use the GPU. In general the GPU may do the job significantly faster. Ensure you have OpenCL  and a valid GPU before enabling this option</string>
        </property>
        <property name="text">
         <string>Use the GPU</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
